libs/clients/national-registry/v3/src/lib/nationalRegistryV3.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@nestjs/common'
+import {
+  EinstaklingarApi,
+  EinstaklingurDTOAllt,
+  EinstaklingurDTOFaeding,
+  EinstaklingurDTOForsjaItem,
+  EinstaklingurDTOHeimili,
+  EinstaklingurDTOHju,
+  EinstaklingurDTOItarAuka,
+  EinstaklingurDTOLogforeldrar,
+  EinstaklingurDTOLoghTengsl,
+  EinstaklingurDTONafnAllt,
+  EinstaklingurDTORikisfang,
+  EinstaklingurDTOTru,
+  GerviEinstaklingarApi,
+} from '../../gen/fetch'
+import { ExcludesFalse } from './utils'
+
+@Injectable()
+export class NationalRegistryV3ClientService {
+  constructor(
+    private individualApi: EinstaklingarApi,
+    private fakeApi: GerviEinstaklingarApi,
+  ) {}
+
+  getAddress = (nationalId: string): Promise<EinstaklingurDTOHeimili | null> =>
+    this.individualApi.midlunV02EinstaklingarNationalIdHeimilisfangGet({
+      nationalId,
+    })
+
+  async getAllDataIndividual(
+    nationalId: string,
+  ): Promise<EinstaklingurDTOAllt | null> {
+    const data = await this.individualApi.midlunV02EinstaklingarNationalIdGet({
```

Comment:
~nit:  you can just return instead of storing in a variable

Reply:
Ah yeah, i was logging something and didn't clean up properly